docker2/driver.Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2018-2020 NVIDIA Corporation
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Released under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+
+# ARG BASE_IMAGE=nvcr.io/nvidia/nvhpc:22.3-devel-cuda_multi-ubuntu20.04
```

Comment:
Should that be something conditional like below
```
if [ ${NVCXX_ENABLED} ]; then
BASE_IMAGE="nvcr.io/nvidia/nvhpc:22.3-devel-cuda_multi-ubuntu20.04"
else
BASE_IMAGE="nvcr.io/nvidia/cuda:11.7.0-devel-ubuntu22.04"
fi;
```

Reply:
There's no conditionals in dockerfiles, it can only be parameterized. 

At least I think.